styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -167,7 +167,7 @@ hr {
 
   .half-padding-full,
   .standard-padding-full {
-    padding-left: $grid-gutter-width / 2;
-    padding-right: $grid-gutter-width / 2;
+    padding-left: $no-side-padding-left;
+    padding-right: $no-side-padding-right;
```

Comment:
I can remove this change and enter it in as a follow up PR. Just keep in mind that it will have padding on the outside edges of these containers. I have no idea when it got added in.